src/Source.jl

```diff
@@ -230,6 +240,7 @@ Keyword Arguments:
 * `append::Bool=false`: if the `sink` argument provided is an existing table, `append=true` will append the source's data to the existing data instead of doing a full replace
 * `transforms::Dict{Union{String,Int},Function}`: a Dict of transforms to apply to values as they are parsed. Note that a column can be specified by either number or column name.
 * `transpose::Bool=false`: when reading the underlying csv data, rows should be treated as columns and columns as rows, thus the resulting dataset will be the "transpose" of the actual csv data.
+* `categorical::Bool=true`: read string columns as CategoricalArrays, as long as the % of unique values seen during type detection isn't 100%
```

Comment:
Need backticks around `CategoricalArray` (a link could be useful too). Also shouldn't 100% be "below 90%"? BTW, maybe that threshold is a bit high? 75% could be more reasonable? Finally, it would be nice to explain that this will reduce memory use dramatically when the number of unique values is small.

Reply:
updated